reverse_test.go

```diff
@@ -28,3 +28,27 @@ func TestReverse(t *testing.T) {
 		})
 	}
 }
+
+func TestCalculator(t *testing.T) {
+	testCalculatorCases := []struct {
+		name string
+		text string
+		want int
+	}{
+		{"Simple string", "Example", 7},
+		{"Russian string", "Россия!!!", 9},
+		{"Chinese string", "你好, 世界。", 7},
+		{"Lots of emojis", "\U0001f60a \U0001f616 \U0001f644 \U0001f925 \U0001f634 \U0001f635 \U0001f480", 13},
+		{"One emoji", "\U0001f60a", 1},
+		{"Empty string", "", 0},
+		{"String with tabs and spaces", "  		", 4},
+		{"Very long string", "oy4iGAPpxkz1NfJd4wak3wAIZ4xfSUFOjlTpLoO1OEQd26vXaBCS23cSnCPPFSOBI0H44T", 70},
+	}
+	for _, tc := range testCalculatorCases {
+		t.Run(fmt.Sprintf("Couldnt count chars in string: %s in case : %s", tc.text, tc.name), func(t *testing.T) {
```

Comment:
What does it print? 
Check the signature of t.Run `func (t *T) Run(name string, f func(t *T)) bool {}` - so what is the first param?